source/components/events/evevent.c

```diff
@@ -312,7 +312,8 @@ AcpiEvFixedEventInitialize (
         {
             Status = AcpiWriteBitRegister (
                 AcpiGbl_FixedEventInfo[i].EnableRegisterId,
-                ACPI_DISABLE_EVENT);
+                (i == ACPI_EVENT_PCIE_WAKE) ?
+		ACPI_ENABLE_EVENT : ACPI_DISABLE_EVENT);
```

Comment:
I think spaces instead of tabs should be used. Please create a fix-up patch.

Reply:
> I think spaces instead of tabs should be used. Please create a fix-up patch.

OK, https://github.com/acpica/acpica/pull/767 created, thanks.